docs/application/native/guides/app-management/app-controls.md

```diff
@@ -808,7 +812,49 @@ The following common application controls are available:
 - [Voice Recorder](common-appcontrols.md#voice-recorder)
 - [VPN Service](common-appcontrols.md#vpn-service)
 
-## Related Information
+<a name="URI"></a>
+## Application control URI
```

Comment:
Everywhere else Application Control C is in caps, only in heading it is in lower case seems inconsistent.
@nawab-samsung can you check on this?

Reply:
thanks @annie-samsung